src/components/overview/feedback/Filelist.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Input } from "semantic-ui-react";
+import { getIPFSIp } from "../../application";
+
+export default class Filelist extends React.Component<any, any> {
+    public viewFeedback = (e, file) => {
+        console.log("feedback: " + file.hash);
```

Comment:
Best not to use console.log but the Blockr-Logger (don't forget the import here after using the suggested code)
```suggestion
        logger.info(JSON.stringify(feedback));
```